apps/penpot-desktop/penpot-desktop.yml

```diff
@@ -0,0 +1,16 @@
+name: Penpot Desktop
+description: A Desktop-Like Experience
```

Comment:
This sounds a bit weird as a description 🤔

Reply:
Okay dokie! I've got something else I use on Product Hunt. Lemme slap it on there.